inst/traduire/en-translation.json

```diff
@@ -69,6 +71,8 @@
     "INDICATOR_LABEL_ANC_ALREADY_ART": "Number already on ART prior prior to first ANC visit",
     "INDICATOR_LABEL_ANC_ART_NEW": "Number of pregnant women initiated on ART at ANC",
     "INDICATOR_LABEL_ANC_KNOWN_POS": "Number of HIV positive ANC attendees aware of status prior to first ANC visit",
+    "INDICATOR_LABEL_ANC_KNOWN_NEG": "Number of HIV negative ANC attendees recently tested and aware of status prior to first ANC visit",
+    "INDICATOR_LABEL_BIRTHS_FACILITY": "Number of health facility births"
```

Comment:
Keys have been added "INDICATOR_LABEL_ANC_KNOWN_NEG" and "INDICATOR_LABEL_BIRTHS_FACILITY" but I cannot see where they are used?

Reply:
I though those were used in the front end somewhere (since they exist for other indicators) - are they not required for time series plots?

Reply:
The other `INDICATOR_LABEL_*` strings are used in the `meta_indicator.csv` which I think is what goes into the output zip

Reply:
Ah I see - will remove.